Development.xcconfig

```diff
@@ -1 +1 @@
-OTHER_SWIFT_FLAGS=$(inherited) -Xfrontend -warn-long-expression-type-checking=400
+OTHER_SWIFT_FLAGS=$(inherited) -DSWIFT_PACKAGE -Xfrontend -warn-long-expression-type-checking=400
```

Comment:
This is a bug that's been resolved upstream. We can probably revert this line in the next beta.

https://bugs.swift.org/browse/SR-7912